functions/src/Models/Elections.ts

```diff
@@ -0,0 +1,25 @@
+import firebase from 'firebase-admin';
+
+export class Elections {
+
+	votingOpen: boolean = false;
+	applicationsOpen: boolean = false;
+	positions = {};
```

Comment:
possibly make this an array. because we then store an array of position objs in there so make this like the User array in the candidate model.